modules/components/pages/processors/http.adoc

```diff
@@ -105,11 +106,15 @@ In order to map or encode the payload to a specific request body, and map the re
 
 == Response codes
 
-Redpanda Connect considers any response code between 200 and 299 inclusive to indicate a successful response, you can add more success status codes with the field `successful_on`.
```

Comment:
Please check this section for accuracy. I've added a new line about redirects and refactored the rest of the paragraph so it is (hopefully) easier to read.